server/views/pages/home.html

```diff
@@ -56,7 +56,7 @@ <h3 class="govuk-heading-s">{{ homePageLinksTitle }}</h3>
         </ul>
       </div>
     </div>
-    {% if config.newDesigns %}
+    {% if config.personalInformation %}
```

Comment:
Really trivial thought: should we carry through the name, to make it more obvious it's a toggle/boolean rather than an object holding personal information?

Handily, naming that would be tricky. `config.enablePersonalInformation` almost looks like it could be a function to enable personal information. `config.PersonalInformationEnabled` might fit better but doesn't match the environment variable name 🤷.

Definitely not blocking for this PR, just an early morning musing 🙂.

Reply:
Completely understand, the latter would be best, maybe a tech debt thing to look at after 😄